File: bungeecord/src/main/java/kr/entree/samplebungee/SpigradleSample.java
package kr.entree.samplebungee;

import kr.entree.spigradle.annotations.PluginMain;
import net.md_5.bungee.api.plugin.Plugin;

/**
 * Created by JunHyung Lim on 2020-06-18
 */
@PluginMain // Annotation is optional, but recommended in many cases
public class SpigradleSample extends Plugin {
    @Override
    public void onEnable() {
        getLogger().info("This is Bungeecord plugin!");
    }
}

File: bungeecord-kotlin/src/main/kotlin/kr/entree/samplebungee/SpigradleSample.kt
package kr.entree.samplebungee

import kr.entree.spigradle.annotations.PluginMain
import net.md_5.bungee.api.plugin.Plugin

/**
 * Created by JunHyung Lim on 2020-06-28
 */
@PluginMain // Annotation is optional, but recommended in many cases
class SpigradleSample : Plugin() {
    override fun onEnable() {
        logger.info("This is Bungeecord plugin!")
    }
}